src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerDetailsController.js

```diff
@@ -0,0 +1,38 @@
+define(['angular', 'application/interviewer/tdprInterviewerModule'], function (angular, tdprInterviewerModule) {
+    tdprInterviewerModule.controller("tdprInterviewerDetailsController", function ($scope, $stateParams, tdprPersonService, $filter, $state, Notification) {
+
+        function init() {
+            tdprPersonService.getPersonDetails($stateParams.id).then(function (response) {
```

Comment:
If we're retrieving person details in every single controller, maybe we should move this to resolve?